doc/iterations/2025.2/add_support_to_dynamically_update_ representation_description.adoc

```diff
@@ -0,0 +1,45 @@
+= Add support to dynamically update representation description
+
+== Problem
+
+A specifier can't update during runtime the description of a representation that was declared programmatically.
+
+We also want to be able to keep the content of a representation while changing its description.
+
+== Key Result
+
+We want to be able to generate a new studio from any representation whose description comes from a view model.
+
+This new studio will be a copy of the current description used by the representation and can be modified.
+
+The representation can then be described by this new studio.
+
+=== Scenario
+
+A specifier wants to change a representation description :
+
+1 - He clicks on the treeItemMenu associated with the representation in the explorer
+
+2 - He clicks on the new option : 'Create a copy of the current description'
+
+3 - He is redirected to a new studio where he can make changes to the representation (a modal was shown to validate the user choice before the redirection)
+
+4 - When he is done, he clicks on the treeItemMenu associated with the description a new button : "publish description" in order to update the representation with the new description
+
+If another user was working on the same representation, one of the following scenario will apply :
+
+1 - The page is refreshed and his representation is updated to the new description.
+
+2 - He can still work with a representation not updated to the new description but the tree item corresponding to the representation shows a different icon or text color to notify him that an update is available, he can then click on the tree item menu "Update" to refresh the representation.
```

Comment:
This would be way more complex than the budget that we have for this improvement. In this work we only want what is currently available for tables but for all representations, that's just it. We will make the whole mechanism more robust and generic but there won't be any major changes in behavior.